src/components/forms/PectraPGRForm.tsx

```diff
@@ -0,0 +1,477 @@
+import {
+  Box,
+  Center,
+  Fade,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Link,
+  Radio,
+  RadioGroup,
+  Stack,
+  useToast
+} from '@chakra-ui/react'
+import { Select } from 'chakra-react-select'
+import { FC } from 'react'
+import { Controller, FormProvider, useForm } from 'react-hook-form'
+import { useRouter } from 'next/router'
+import { zodResolver } from '@hookform/resolvers/zod'
+
+import { DropdownIndicator, PageText } from '../UI'
+import { SubmitButton } from '../SubmitButton'
+import { Captcha, Field, TextAreaField, TextField, UploadFile } from '.'
+
+import { api } from './api'
+
+import { chakraStyles } from './selectStyles'
+
+import {
+  COUNTRY_OPTIONS,
+  HOW_DID_YOU_HEAR_ABOUT_GRANTS_WAVE,
+  INDIVIDUAL,
+  OTHER,
+  PECTRA_PGR_PROJECT_CATEGORY_OPTIONS,
+  TEAM,
+  TIMEZONE_OPTIONS,
+} from './constants'
+
+import { PECTRA_PGR_THANK_YOU_PAGE_URL, TOAST_OPTIONS } from '../../constants'
+
+import { PectraPGRSchema, PectraPGRData } from './schemas/PectraPGR'
+
+export const PectraPGRForm: FC = () => {
+  const router = useRouter()
+  const toast = useToast()
+
+  const methods = useForm<PectraPGRData>({
+    mode: 'onBlur',
+    shouldFocusError: true,
+    defaultValues: {
+      individualOrTeam: INDIVIDUAL,
```

Comment:
repeatApplicant needs a default value because it is a required field
```suggestion
      individualOrTeam: INDIVIDUAL,
      repeatApplicant: false
```